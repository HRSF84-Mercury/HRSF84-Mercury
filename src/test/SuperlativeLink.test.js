import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import SuperlativeLink from '../components/SuperlativeLink';

describe('SuperlativeLink', () => {
  it('should render correctly with props', () => {
    const superlative = {
      'img':'https://www.w3schools.com/w3css/img_fjords.jpg',
      'superlative':'This is a really cool superlative'
    };
    const output = shallow(
      <SuperlativeLink superlative={superlative} />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});

