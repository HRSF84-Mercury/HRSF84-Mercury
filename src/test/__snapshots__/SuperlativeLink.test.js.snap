// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuperlativeLink should render correctly with props 1`] = `
<a
  className="superlative-details-info container column"
  href="/superlative/This is a really cool superlative"
>
  <img
    className="superlative-img"
    src="https://www.w3schools.com/w3css/img_fjords.jpg"
  />
  <div
    className="container column"
  >
    <div
      className="superlative-name"
    >
      This is a really cool superlative
    </div>
  </div>
</a>
`;
